product/roundhouse.console/Program.cs

```diff
@@ -429,6 +432,12 @@ private static void run_diff_utility(ConfigurationPropertyHolder configuration)
             }
         }
 
+        public static void init_security_protocol()
+        {
+            // allow tls
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12 | SecurityProtocolType.Ssl3;

```

Comment:
Just confirmed it's definitely not there in .NET 4.0.

`SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12`

Here is 3.5 - https://msdn.microsoft.com/en-us/library/system.net.securityprotocoltype(v=vs.90).aspx
Here is 4.0 - https://msdn.microsoft.com/en-us/library/system.net.securityprotocoltype(v=vs.100).aspx